.github/workflows/test-holonix-setup.yml

```diff
@@ -80,12 +80,14 @@ jobs:
             echo Failed as expected.
           }
 
-          # Now actually configure the cache and try a build
+          # Now actually configure the cache
           bash <(curl https://raw.githubusercontent.com/holochain/holochain/${{ github.sha }}/configure-cache.sh) use holochain-ci
-          do_nix_build
           
           # Give the daemon a chance to restart
-          sleep 1
+          sleep 5
+          
+          # and try a build where everything is expected to be cached
+          do_nix_build
```

Comment:
could we use the nix-check-cache here? if not, we should probably adapt that one to be usable here as that it's purpose.

Reply:
Yes we absolutely could use it, there's nothing in it that would struggle here, just need a way to grab the binary. I'll stick in a TODO for now

Reply:
wouldn't a separate action step work as it's already exposed as an action?